backend/src/schema/resolvers/posts.js

```diff
@@ -77,14 +77,16 @@ export default {
   Mutation: {
     CreatePost: async (_parent, params, context, _resolveInfo) => {
       const { categoryIds } = params
+      const { image: imageInput } = params
       delete params.categoryIds
-      params = await fileUpload(params, { file: 'imageUpload', url: 'image' })
+      delete params.image
       params.id = params.id || uuid()
       const session = context.driver.session()
       const writeTxResultPromise = session.writeTransaction(async transaction => {
         const createPostTransactionResponse = await transaction.run(
           `
-            CREATE (post:Post {params})
+            CREATE (post:Post)
+            SET post += $params
```

Comment:
why should this be `+=` ?? 
it's creating a post, so it should never have any `params` to add to, oder?

Reply:
You're right. But it does not hurt either, or does it? :thinking: